Jenkinsfile-Release

```diff
@@ -59,9 +59,15 @@ dockerizedBuildPipeline(
     runSafely "git checkout ${gitBranch(env)}"
     if ('nexus-iq-server'.equals(params.chart)) {
       runSafely "./upgrade.sh charts/nexus-iq ${chartVersion} ${params.appVersion}"
-      runSafely './build.sh'
+      runSafely './build-iq.sh'
       runSafely 'git add charts'
     }
+
+    if ('nexus-repository-manager'.equals(params.chart)) {
+      runSafely "curl -L 'https://github.com/sonatype/nxrm3-helm-repository/blob/main/docs/${nexusRepoManagerArtifactName}?raw=true' -o 'docs/${nexusRepoManagerArtifactName}'"
+      runSafely "curl -L 'https://github.com/sonatype/nxrm3-helm-repository/blob/main/docs/${awsResiliencyArtifactName}?raw=true' -o 'docs/${awsResiliencyArtifactName}'"
+      runSafely './build-nexus.sh'
+    }
```

Comment:
this block also exists below (line ~75 or so). can it be removed from there?

Reply:
Ah yes, that needs to come out

Reply:
https://github.com/sonatype/helm3-charts/pull/245/commits/70dde6c411cfee6d1e51f5e5e455ccdbe2ac0052